docs/services/navigation/_index.md

```diff
@@ -554,3 +554,119 @@ obstacles = await my_nav.get_obstacles()
 
 {{% /tab %}}
 {{< /tabs >}}
+
+## Concepts
+
+### CompassHeading
+
+Compass heading measurements are read by the following {{< glossary_tooltip term_id="model" text="models" >}} of [movement sensor](/components/movement-sensor/):
+
+- [gps-nmea](/components/movement-sensor/gps/gps-nmea/)
+- [gps-nmea-rtk-pmtk](/components/movement-sensor/gps/gps-nmea-rtk-pmtk/)
+- [gps-nmea-rtk-serial](/components/movement-sensor/gps/gps-nmea-rtk-serial/)
+
+An example of a `Compass Heading` reading:
+
+``` go
+gps.GetCompassHeading{175}
+```
+
+Compass heading readings are taken through the movement sensor API's [`GetCompassHeading()`](/components/movement-sensor/#getcompassheading).
+If you want to read compass headings, [configure a capable movement sensor](/components/movement-sensor/#configuration) on your robot.
+Use compass heading readings to determine the *bearing* of your robot, or, the [cardinal direction](https://en.wikipedia.org/wiki/Cardinal_direction) that your robot is facing.
```

Comment:
The wit and gps give you compass heading, but they're hacky - we're looking for a driver that gives you compass heading, this is more to do with finding good hardware that gives you a stable compass heading rather than the API. 

I'd mention the wit though, anyway.

Reply:
good w that! Done